doc/nrf/protocols/matter/overview/bridge.rst

```diff
@@ -0,0 +1,63 @@
+.. _ug_matter_overview_bridge:
+
+Matter Bridge
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter bridge is a device that allows to use non-Matter IoT devices in a Matter fabric together with other native Matter devices.
+The non-Matter devices in such configuration are called bridged devices.
+It is out of the Matter specification scope to define connectivity technologies used by the bridged devices, so they can use any technology (for example Bluetooth LE or Zigbee), as long as bridge device supports it too.
+
+The goal of a bridge is to expose the bridged devices to Matter nodes in a fabric, as a Matter endpoints.
+The exposure shall be transparent, that is the Matter nodes should be able to communicate with non-Matter bridged devices as they would be native Matter devices.
+
+The following figure illustrates network topology example with Matter controller using Matter light bulb and bridged Bluetooth LE light bulb as Matter endpoints, thanks to the Matter bridge device that is bridging the Bluetooth LE device to Matter fabric:
+
+.. figure:: images/matter_bridge_topology.svg
+   :alt: Matter topology using bridged Bluetooth LE device example
+
+   Matter topology using bridged Bluetooth LE device example
+
+Bridging principles
+*******************
+
+The Matter bridge uses a dedicated Matter endpoint fulfilling the Aggregator device type requirements to represent bridging capabilities.
+Especially, this endpoint uses Descriptor cluster with PartsList attribute to store all endpoints representing the bridged devices connected to the bridge.
+
+Additionally, the bridge represents every bridged device using one or more Matter endpoints, depending on the functionalities supported by the device.
+Every bridged device exposed by the bridge has to fulfill all requirements for corresponding Matter device type.
+Unlike native Matter nodes, the bridged device has dedicated Bridged Device Basic Information cluster enabled on its endpoint to provide basic information about the device.
+
+Bridged devices can be added to bridge or removed from it at runtime, so the list of endpoints exposed by the bridge may be dynamically updated.
+
+Implementation in |NCS|
+***********************
+
+The |NCS| delivers reference implementation of the Matter bridge core that is located in the :file:`samples/matter/common/bridge` directory.
+The core implementation provides basic bridge functionalities and generic API that is not dependent on any specific connectivity technology.
+Beside of the core part, the |NCS| also delivers example `matter_bridge_app` that can be used as a reference for creating own application.
```

Comment:
Link to the app README page?

Reply:
Oh right, I thought `matter_bridge_app` will work as a link, but it didn't. I will fix it.